### PR TITLE
feat: tsed platform migration

### PR DESCRIPTION
**Pull request by axejintao**
# Summary

- upgrade to `platform-serverless` as older packages are deprecated
- begin migration to removal of all `@tsed/common` references as suggested by library documentation 
- initial ideas for global test chain harness

cc: @NamelsKing @mrbasado on requirements from removals of references to common library

### CODE REVIEW

File: src/chains/config/chain.config.ts [hunk 1]
```diff
@@ -93,16 +80,16 @@ export abstract class Chain {
   }
 
   async getSdk(): Promise<BadgerSDK> {
-    const sdk = Chain.sdks[this.network];
-    await sdk.ready();
-    return sdk;
+    await this.sdk.ready();
+    return this.sdk;
```
Comment (NamelsKing) [inline src/chains/config/chain.config.ts:84]: ye, this is nice one 😃

File: src/config/logger/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { $log, PlatformLoggerSettings } from '@tsed/common';
+
+import { isProduction } from '../envs/index';
+
+if (isProduction) {
```
Comment (NamelsKing) [inline src/config/logger/index.ts:5]: convenient to have in dev to, imho

File: src/handler.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { PlatformServerless } from '@tsed/platform-serverless';
+
+import { V2_CONTROLLERS, V3_CONTROLLERS } from './controllers';
+// import { swaggerConfig } from './config/constants';
```
Comment (NamelsKing) [inline src/handler.ts:4]: we dont support swagger anymore?

**Review by NamelsKing — CHANGES_REQUESTED** [review]

File: src/handler.ts (hunk 3 above)
Comment (axejintao) [reply src/handler.ts:4]: we will need to re enable swagger or figure out how it fits in the new platform. we will not release up directly until yeah we support it again.

File: src/config/logger/index.ts (hunk 2 above)
Comment (axejintao) [reply src/config/logger/index.ts:5]: sure, this was added by tsed cli - not sure what it does yet lol

**Review by NamelsKing — APPROVED** [review]

**Pull request merged**